client/src/components/customers.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+
+class Customers extends Component {
+  constructor(){
+    super();
+    this.state={
+      customers:[]
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>Hello World</h1>
```

Comment:
let's render the data that we got into the website, so we can have that step done and check if everything fully working, it could be changed later thats not a biggie

Reply:
Done